src/d3a/d3a_core/redis_connections/redis_communication.py

```diff
@@ -35,12 +37,21 @@
 RESULTS_CHANNEL = "d3a-results"
 ZIP_RESULTS_CHANNEL = "d3a-zip-results"
 ZIP_RESULTS_KEY = "d3a-zip-results-key/"
+HEARTBEAT_CHANNEL = "d3a-heartbeat"
+PULSE_RATE = 5  # in secs
```

Comment:
It would be even better if this was part of the constants.py. Or, if this value is used from the d3a-web too, in d3a-interface.